### DISCUSSION TITLE
2009/05/13/%E3%80%8A%E7%AC%AC%E5%8D%81%E4%B8%89%E6%A5%BC%E3%80%8B%E8%A7%82%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#259] [Announcements]
# 2009/05/13/%E3%80%8A%E7%AC%AC%E5%8D%81%E4%B8%89%E6%A5%BC%E3%80%8B%E8%A7%82%E5%90%8E%E6%84%9F

最近虽然工作时间还是一样忙，不过下班总算不用再写书了，所以一下子轻松不少。终于可以安安心心的看几部电影了。电影中我最喜欢看的是科幻电影，我喜最欣赏想象力丰富的故事，科幻电影通常都充满各种奇思妙想。昨天看了一部影片叫做《The Thirteenth Floor》的电影，中文翻译为《异次元骇客》或《第十三楼》。

https://blog.qizhen.xyz/2009/05/13/%E3%80%8A%E7%AC%AC%E5%8D%81%E4%B8%89%E6%A5%BC%E3%80%8B%E8%A7%82%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

风
2009年05月25日 10:01 编辑
智慧的代价就是矛盾，这是人生对人生观开的一个玩笑。

回复
杰
2009年05月16日 16:00 编辑
幸福就是无知，呵呵。maybe.

回复
Haiying
2009年05月14日 22:05 编辑
你让我想起，诺丁山里那个被介绍的女友，看人吃煮熟的胡萝卜，表情无限悲伤的说"that\’s been murdered"

回复
Percy
2009年05月14日 08:35 编辑
原来你也是这么感性的…任何的一切在死亡面前都显得那么微不足道，所以人们往往也只是在谈及死亡、面对死亡的时候才会感叹和珍惜。当自己回到现实，却依旧被各种俗事所困扰，想不开。人终究是俗人。

回复
Qizhen
2009年05月14日 03:54 编辑
我吃饭的时候，有时候会想：养殖场里的猪们是否知道自己的命运的，如果它们知道自己生来就是准备被吃掉的会不会很痛苦？如果人类得知自己只是被“上帝”饲养的食品，会不会也很痛苦。我估计大多数人不会特别痛苦的，就好像现在每个人都知道自己迟早会死，但这并不影响人们追寻活着的快乐。如果命运不能被改变，那最好试着去享受它;)

回复
Haiying
2009年05月13日 22:10 编辑
sounds very interesting. I will look for this film

回复
yi
2009年05月13日 21:33 编辑
恭喜恭喜！好久没来了，以前在学校遇到Labview问题就会上你的主页，呵呵。期待你的书，祝你成功！

回复
